docs/docs/sourcing-from-builder-io.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Sourcing from Builder.io
+---
+
+[Builder.io](https://builder.io) has all the benefits of a modern headless CMS platform, plus the power of an easy to learn drag and drop editor that enables everyone to edit more than just content. Builder + Gatsby empowers your entire team to create performant and fully customizable pages quickly.
+
+<img src="https://imgur.com/HjBWIbv.gif" alt="Editor example" />
+
+---
+
+## Quick Start
+
+### Fresh project:
+
+To start on a fresh project quickly, take a look at our [Gatsby starter](https://github.com/BuilderIO/gatsby-starter-builder)
+
+```sh
+gatsby new my-builder-site https://github.com/BuilderIO/gatsby-starter-builder
+```
+
+### Existing project:
+
+Make a free account over at [Builder.io](https://www.builder.io/) and grab your public API key from your [organization page](https://builder.io/account/organization) and:
+
+```javascript
+// In your gatsby-config.js
+const path = require("path")
+module.exports = {
+  plugins: [
+    {
+      resolve: "@builder.io/gatsby",
+      options: {
+        // public API Key
+        publicAPIKey: "MY_PUBLIC_API_KEY",
```

Comment:
After setting up the starter, it suggests that the API key be placed in the `config.js` and is called here as `config.builderAPIKey`. This code block suggests otherwise. Can you clarify ?

Reply:
Hi @Ekwuno thanks for reviewing, this instruction for adding BuilderIO to an existing project in this case you only need the plugin, while the starter is to make it easy to start on a fresh project.

In the starter the `config.js` is just an abstraction so this apikey will be passed down to the @builder.io/gatsby plugin here

https://github.com/BuilderIO/gatsby-starter-builder/blob/e7985bf58ccdb85a1b75b5d7355b3a2a55ec7875/gatsby-config.js#L28

And the same key will be used to initialize `builder` instance here
https://github.com/BuilderIO/gatsby-starter-builder/blob/e7985bf58ccdb85a1b75b5d7355b3a2a55ec7875//src/builder-settings.js#L10